renovate.json

```diff
@@ -1,7 +1,30 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:recommended",
-    "github>redhat-exd-rebuilds/renovate-config"
+  "baseBranches": ["main"],
+  "prConcurrentLimit": 3,
+  "lockFileMaintenance": {
+    "enabled": false
+  },
+  "postUpdateOptions": ["gomodTidy"],
```

Comment:
do we also need gomodVendor ?

Reply:
AFAIK if renovate finds a vendor folder, it always vendor dependencies